sdcm/utils/issues.py

```diff
@@ -28,6 +30,26 @@ class Issue:
     issue_id: int | None
 
 
+class CachedGitHubIssues:
```

Comment:
I'd add docstring how cache is populated - reference to gh action.

Reply:
added one